src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -101,7 +101,7 @@ grounded_dummy("dummy_var").
         std::make_tuple(ManifestDatalogFacts({ManifestDatalogFacts::Particle(
                             particle_spec.get(), {}, {})}),
                         *(AuthorizationLogicDatalogFacts::create(
-                            AuthorizationLogicTest::GetTestDataDir(),
+                            raksha::test_utils::GetTestDataDir("src/xform_to_datalog/testdata"),
```

Comment:
It looks like you're using the same string literal twice here ("src/xform_to_datalog/testdata"). To prevent it from being edited in one place and not the other, it might be nice to make it a `const static absl::string_view` global.